### ISSUE TITLE
Remove quaternions

### ISSUE THREAD

**Issue by janden** [ComputationalCryoEM/ASPIRE-Python#194]
These seem have snuck in from the old Matlab code. Specifically the functions `qrand`, `qrand_rots`, and `q_to_rot` should be removed. The only use case is generating random rotations, which can be done using `uniform_random_angles` and converting from Euler angles using `scipy.spatial.transform.Rotation`.

**Comment by junchaoxia** [comment]
@janden Those quaternions function were implemented to reproduce the cov2d example from the Matlab version. I took a look today. We need to remove them from the Cov2D example and unit tests. Of course the unit tests need to be rebuilt. That can generate conflicts with @garrettwrong work. Maybe postpone this?

**Comment by janden** [comment]
Ok let's leave it for now and come back to it once the row-major stuff is merged.

**Comment by junchaoxia** [comment]
This issue was fixed in PR 245. Closing now.
